storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst

```diff
@@ -35,28 +35,33 @@ Setting up a slave for replication with |Percona XtraBackup| is really a very st
 * ``Percona XtraBackup``
   The backup tool we will use. It should be installed in both computers for convenience.
 
+.. note:: It is not recommended to mix MySQL variants
+   (Percona Server, MySQL, MariaDB) in your replication setup.
+   This may produce incorrect :file:`xtrabackup_slave_info` file
+   when adding a new slave.
+
 STEP 1: Make a backup on ``TheMaster`` and prepare it
 =====================================================
 
 At ``TheMaster``, issue the following to a shell:
 
 .. code-block:: console
 
-   TheMaster$ innobackupex --user=yourDBuser --password=MaGiCdB1 /path/to/backupdir 
+   TheMaster$ xtrabackup --user=yourDBuser --password=MaGiCdB1 /path/to/backupdir 
 
 After this is finished you should get:
 
 .. code-block:: console
 
-   innobackupex: completed OK! 
+   xtrabackup: completed OK! 
 
-This will make a copy of your |MySQL| data dir to the /path/to/backupdir/$TIMESTAMP. You have told |Percona XtraBackup| (through the |innobackupex| script) to connect to the database server using your database user and password, and do a hot backup of all your data in it (all |MyISAM|, |InnoDB| tables and indexes in them).
+This will make a copy of your |MySQL| data dir to the /path/to/backupdir/$TIMESTAMP. You have told |Percona XtraBackup| to connect to the database server using your database user and password, and do a hot backup of all your data in it (all |MyISAM|, |InnoDB| tables and indexes in them).
 
 In order for snapshot to be consistent you need to prepare the data:
 
 .. code-block:: console
 
-   TheMaster$ innobackupex --user=yourDBuser --password=MaGiCdB1 \
+   TheMaster$ xtrabackup --user=yourDBuser --password=MaGiCdB1 \
```

Comment:
xtrabackup --user=yourDBuser --password=MaGiCdB1 --prepare --target-dir=/path/to/backupdir